src/sql/platform/capabilities/common/capabilitiesService.ts

```diff
@@ -21,6 +21,7 @@ export interface ConnectionProviderProperties {
 	providerId: string;
 	displayName: string;
 	azureResource?: string;
+	isLanguageFlavorProvider?: boolean;
```

Comment:
Are there any connection providers that are not language flavor providers? I thought every connected editor would have the associated language dialect selected in the drop-down. Maybe we can get away without this extra property? Otherwise we will need to add it to MySQL extension too...

Reply:
Yes there is one MSSQL-CMS which is not a flavor provider. So adding this property in extension to distinguish.